### PR TITLE
[Merged by Bors] - chore: typos in doc strings

### PR DESCRIPTION
**Pull request by mo271**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/CategoryTheory/Abelian/InjectiveResolution.lean [hunk 1]
```diff
@@ -322,7 +322,7 @@ lemma ofCocomplex_d_0_1 :
     (ofCocomplex Z).d 0 1 = d (Injective.ι Z) := by
   simp [ofCocomplex]
 
-#adaptation_note /-- Since nightly-2024-03-11, this takes takes forever now -/
+#adaptation_note /-- Since nightly-2024-03-11, this takes forever now -/
```
Comment (grunweg) [inline Mathlib/CategoryTheory/Abelian/InjectiveResolution.lean:325]: Also remove the "now"? (Also, possibly out of scope : is this still current?)

File: Mathlib/LinearAlgebra/Matrix/Spectrum.lean [hunk 2]
```diff
@@ -85,7 +85,7 @@ theorem star_eigenvectorUnitary_mulVec (j : n) :
     (star (eigenvectorUnitary hA : Matrix n n 𝕜)) *ᵥ ⇑(hA.eigenvectorBasis j) = Pi.single j 1 := by
   rw [← eigenvectorUnitary_mulVec, mulVec_mulVec, unitary.coe_star_mul_self, one_mulVec]
 
-/--Unitary diagonalization of a Hermitian matrix.-/
+/-- Unitary diagonalization of a Hermitian matrix.-/
```
Comment (grunweg) [inline Mathlib/LinearAlgebra/Matrix/Spectrum.lean:88]:
```suggestion
/-- Unitary diagonalization of a Hermitian matrix. -/

**Review by grunweg — COMMENTED** [review]
Thanks, just one nit and one question.

**Review by grunweg — APPROVED** [review]

File: Mathlib/CategoryTheory/Abelian/InjectiveResolution.lean (hunk 1 above)
Comment (mo271) [reply Mathlib/CategoryTheory/Abelian/InjectiveResolution.lean:325]: I haven't checked

**Comment by grunweg** [comment]
Thanks for this clean-up!
maintainer merge

**Pull request closed without merging**